### ISSUE TITLE
WCAG Compliance

### ISSUE THREAD

**Issue by SweetlyBossy** [SweetlyBossy/FED-PE1-SaraAL#10]
### GOAL:
Ensure the web application is accesible and inclusive for all users by achieving full compliance with WCAG 2.1 standards. This builds the users trust and guarentees access for individuals with visual disabilities.

### RECOMMENDED TOOLS
[WAVE](https://wave.webaim.org/)
[Google Lighthouse](https://chromewebstore.google.com/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk)

### TECHNICAL REQUIREMENTS
- [x] Alternative text.
- Make sure all images, logos, and icons have a descriptive alt text on them,
- [x] Correct use of semantics
- Make sure to use the correct semantic on the HTML.
- [x] Correct amount of contrast
- LARGE TEXT - 3 : 1 ratio.
- Body text - 4.5 : 1 ratio
- Make sure that the text colors, and backgrounds colors do make it hard to see and read.
- [x] Accessible forms.
- Label every input field in a form.
- Display a user-friendly error message when the input validation fails.

### ACCEPTANCE CRITERIA
- [x] The web app must comply with WCAG 2.1 standard without compromising the functionality.
- [x] All identified accessibility issues must be resolved, without effecting usability.
- [x] Provide informative, accessible error feedback, when applicable.
- [x] Ensure full responsiveness across all screen and devices.
